src/cmd-buildextend-live

```diff
@@ -524,6 +524,14 @@ def generate_iso():
             if len(vendor_ids) != 1:
                 raise Exception(f"did not find exactly one EFI vendor ID: {vendor_ids}")
 
+            # always replace EFI/{vendor}/grub.cfg to actual vendor_id[0]
+            # https://github.com/openshift/os/issues/954
+            grubfilepath = ensure_glob(os.path.join(tmpdir, 'live/EFI/*/grub.cfg'))
```

Comment:
I can not catch the meaning about `rename the entire directory`, what I do currently is rename `/srv/data/c9s/tmp/buildpost-live/live/EFI/redhat/grub.cfg --> /srv/data/c9s/tmp/buildpost-live/live/EFI/{vendor_ids[0]}/grub.cfg`, could you help to clarify? Thanks!

Reply:
Currently `grub.cfg` is the only file in `openshift/os` `live/EFI/redhat`.  If we add other files later, those should also be moved from `redhat` to `{vendor_ids[0]}`.  So it'd be better for this code to move everything in the `redhat` directory instead of only `grub.cfg`.

Reply:
So should rename `/srv/data/c9s/tmp/buildpost-live/live/EFI/redhat --> /srv/data/c9s/tmp/buildpost-live/live/EFI/{vendor_ids[0]}/`, is this correct?